store/src/java/com/zimbra/cs/service/admin/DeleteAccount.java

```diff
@@ -76,6 +74,7 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
         ZimbraSoapContext zsc = getZimbraSoapContext(context);
         Provisioning prov = Provisioning.getInstance();
 
+        prov.validateLicense();
```

Comment:
No.. I did not tell you to this..

Reply:
Made changes as per our last discussion.